srcalc/src/main/webapp/WEB-INF/views/fragments/editVariableKey.jspf

```diff
@@ -0,0 +1,13 @@
+<%--
+A simple JSP fragment to edit a variable key. This content expects to be in a
+Spring <form> tag with the form object of type EditVariable and called
+'variable'.
```

Comment:
I think this comment should also mention that it should be inside of a <table> tag.